test/built-ins/Function/prototype/toString/built-in-function-object.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-function.prototype.tostring
+description: >
+  toString of built-in Function object
+info: |
+  ...
+  If func is a Bound Function exotic object or a built-in Function object, then return an implementation-dependent String source code representation of func.
+  The representation must have the syntax of a NativeFunction.
+  ...
+
+  NativeFunction:
+    function IdentifierName_opt ( FormalParameters ) { [ native code ] }
+
+includes: [fnGlobalObject.js, nativeFunctionMatcher.js, wellKnownIntrinsicObjects.js]
+---*/
+
+var visited = [];
+var verified = [];
+
+function visit(object, assertion) {
+  if (visited.includes(object)) {
+    return;
+  }
+
+  visited.push(object);
+
+  if (typeof object === "function") {
+    assertion(object);
+    verified.push(object.name);
```

Comment:
...so in this case it will iterate over each case for the props:

- intrinsicName
- globalNameOrSource
- reference

but what you want is probably the .reference only.

Reply:
I don't understand what you mean by this.